### PR TITLE
Adding linting to the guidelines

### PR DESCRIPTION
**Pull request by rkreutz-teamwork**
# About

This PR adds information about linting our swift projects.

### CODE REVIEW

**Comment by josealobato** [comment]
Oh yes, this is great stuff Rodrigo, thanks!

I do not know if this is the right place but, should we here suggest the strategy to follow on the implementation and usage of the tool? Like:

* The configuration file should be placed in `bla` and included in control version / or should be local and excluded from it.
* Lint execution should be done on the build phase by adding the corresponding script  `bla bla bla` / or should be used in command line.

At the end is a guideline.

what do you think?

**Review by josealobato — APPROVED** [review]

File: README.markdown [hunk 1]
```diff
@@ -1094,6 +1095,97 @@ In larger expressions, optional parentheses can sometimes make code read more cl
 ```swift
 let playerMark = (player == current ? "X" : "O")
 ```
+## Linting
+
+ We use SwiftLint to lint our projects so we guarantee our guidelines are being enforced during development. For more details on how SwiftLint works and how to install it please [check this](https://github.com/realm/SwiftLint).
+
```
Comment (dfreniche) [inline README.markdown]: How are we using SwiftLint in our projects? As a brew binary or integrated as a pod? I'll add this info. Also, a TL;DR; of the installation method (`brew install swiftlint` or `pod install`)

**Comment by dfreniche** [comment]
I second @josealobato's comment. The idea is: if a new dev joins the team, and has never used SwiftLint, can this person configure the linter in a new project easily reading this notes? I know, I know, [RTFM](https://en.wikipedia.org/wiki/RTFM) Swiftlint, but if we can help next projects start faster...

**Review by dfreniche — CHANGES_REQUESTED** [review]
Just a bit more detail. Other than that: great!

**Comment by dfreniche** [comment]
Changed branch to abandon gitflow and only use `master` per: https://digitalcrew.teamwork.com/#/tasks/12739917?c=5855155&

File: README.markdown (hunk 1 above)
Comment (rkreutz-teamwork) [reply README.markdown]: Done b3de5ec

**Review by NSLeoFont — APPROVED** [review]

**Review by dfreniche — APPROVED** [review]
LGTM

**Pull request merged**